kitsune/search/v2/tests/test_documents.py

```diff
@@ -0,0 +1,42 @@
+from kitsune.sumo.tests import TestCase
```

Comment:
nit: How about `test_annotations` so that we can categorize tests per operation? Otherwise we will end up with a lot of tests here

Reply:
IMO we should wait until that becomes a problem until we reorganise the tests